main.go

```diff
@@ -12,6 +12,10 @@ import (
 
 func main() {
 	ctx := context.Background()
+	source, err := settings.NewEnvSource(os.Environ())
```

Comment:
Question: is the configuration of the device42 IPAMFetcher components coming in another PR?

Reply:
:+1: yes, it is fairly involved from a first pass, so I will break that work into another PR